web/client/plugins/map/selector.js

```diff
@@ -24,7 +24,8 @@ export default createShallowSelectorCreator(isEqual)(
     isLocalizedLayerStylesEnabledSelector,
     localizedLayerStylesNameSelector,
     currentLocaleLanguageSelector,
-    (projectionDefs, map, mapType, layers, features, loadingError, securityToken, elevationEnabled, isLocalizedLayerStylesEnabled, localizedLayerStylesName, currentLocaleLanguage) => ({
+    mapNameSelector,
```

Comment:
no tests for this ?

Reply:
test for what exactly ? there is already a unit test for 'mapNameSelector' here: web\client\plugins\map\__tests__\selector-test.jsx

Reply:
for createShallowSelectorCreator that you have extended